app/src/main/java/de/danoeh/antennapod/fragment/SearchFragment.java

```diff
@@ -73,16 +78,18 @@ public static SearchFragment newInstance(String query) {
         Bundle args = new Bundle();
         args.putString(ARG_QUERY, query);
         args.putLong(ARG_FEED, 0);
+        args.putString(FEED_TITLE, "");
         fragment.setArguments(args);
         return fragment;
     }
 
     /**
      * Create a new SearchFragment that searches one specific feed.
      */
-    public static SearchFragment newInstance(String query, long feed) {
+    public static SearchFragment newInstance(String query, long feed, String feedTitle) {
         SearchFragment fragment = newInstance(query);
         fragment.getArguments().putLong(ARG_FEED, feed);
+        FEED_TITLE = feedTitle;
```

Comment:
Please add an argument instead of using a static variable. You can transfer the title just like the ID in the line above.